### PR TITLE
Testing/route details

### PR DESCRIPTION
**Pull request by PaigeVannelli**
## Is this a fix or a feature?
Testing

## What is the change?
Add testing for matched routes user flow and details. Add new profile photos and randomize displayed photo. 

## What does it fix?
n/a

## Where should the reviewer start?
review ProfilePhoto component and hitch-detailed-route-spec

## How should this be tested?
run hitch-detailed-route-spec tests to see testing coverage

### CODE REVIEW

File: cypress/fixtures/matched-rides.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+          "user_name": "name",
+          "zipcode_origin": "80211", 
+          "zipcode_destination": "80017" 
+        }
+      ],
+      "id": "1",
+      "type": "matched"
+    }
+  }
+}
```
Comment (itsnameissteven) [inline cypress/fixtures/matched-rides.json:36]: nice, fixture.

**Review by itsnameissteven — APPROVED** [review]
Everything looks good. Ready to merge after everyone reviews

File: cypress/fixtures/matched-rides.json (hunk 1 above)
Comment (alexthompson207) [reply cypress/fixtures/matched-rides.json:36]: agreed!

File: src/apiCalls.ts [hunk 2]
```diff
@@ -46,7 +46,6 @@ export const getUserByID = (id: number) => {
 
 
 const handleErrors = (response: Response) => {
-  console.log(response)
```
Comment (alexthompson207) [inline src/apiCalls.ts:49]: Nice job removing console.logs!

**Review by alexthompson207 — APPROVED** [review]
Nice work with the testing! Also love the random profile pictures, great work!

**Pull request merged**